inc/namespace.php

```diff
@@ -954,3 +960,43 @@ function set_wp_debug_constants() : void {
 	}
 	define( 'WP_DEBUG_DISPLAY', false );
 }
+
+/**
+ * Log a message to our cloud services. This will use Fluent Bit if it is
+ * available, otherwise log directly to CloudWatch..
+ *
+ * @param string $log_group Name of the log group to send logs to. Environment
+ *                          name is added automatically.
+ * @param string $log_stream Name of the log stream.
+ * @param string $message Message to log
+ * @param string $level Level of the log. Corresponds to Psr\Log\LoggerInterface
+ * @see https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-3-logger-interface.md#3-psrlogloggerinterface
+ * @return boolean
+ */
+function log_to_cloud( string $log_group, string $log_stream, string $message, string $level = 'info' ) : bool {
```

Comment:
Wondering if there's a better function name here, @rmccue for ideas perhaps.

Reply:
Yeah, totally open to better ideas.

Reply:
Considering we disallow `use function`, it could simply be called `log()`? It'd be used as `Cloud\log()`.

Reply:
We'll use `Cloud\get_logger()` instead.